.github/workflows/test.yml

```
@@ -55,6 +55,44 @@ jobs:
+      run: uv python install 3.12
+
+    - name: Install dependencies
+      run: |
+        uv sync --all-extras
+
+    - name: Install Playwright Chromium
+      run: |
+        uv run python -m playwright install --with-deps chromium
+
+    - name: Set environment variables for dashboard tests
+      run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:47: Double quote to prevent globbing and word splitting

(shellcheck)